src/hotspot/share/opto/addnode.cpp

```diff
@@ -191,10 +191,9 @@ Node *AddNode::Ideal(PhaseGVN *phase, bool can_reshape) {
       set_req(1, addx);
       set_req(2, a22);
       progress = this;
-      PhaseIterGVN *igvn = phase->is_IterGVN();
-      if (add2->outcnt() == 0 && igvn) {
+      if (add2->outcnt() == 0 && phase->is_IterGVN()) {
```

Comment:
Why did you remove the local variable `igvn`? Performance wise it shouldn't make a difference, right?